api/tests/devcomp/integration/repositories/module-repository_test.js

```diff
@@ -100,6 +100,47 @@ describe('Integration | DevComp | Repositories | ModuleRepository', function ()
   });
 
   describe('#list', function () {
+    describe('errors', function () {
+      describe('if there are duplicate IDs in modules content', function () {
+        it('should throw an Error', async function () {
```

Comment:
J'aurais inversé les conditions :
```suggestion
describe('if there are no duplicated IDs in modules content', function () {
        it('should result an empty array of duplicated ids', async function () {
```